simulariumio/data_objects/meta_data.py

```diff
@@ -43,14 +50,29 @@ def __init__(
             A multiplier for the scene, use if
             visualization is too large or small
             Default: 1.0
+        trajectory_title : str (optional)
+            A title for this run of the model
+        model_meta_data: ModelMetaData (optional)
+            Metadata for the model that produced this trajectory
         """
+        # box_size defaults to None here so that later,
+        # when it's used to override box_size in data,
+        # it's easy to test whether the user has specified it
         self.box_size = box_size
-        self.camera_defaults = camera_defaults
+        self.camera_defaults = (
+            camera_defaults if camera_defaults is not None else CameraData()
+        )
         self.scale_factor = scale_factor
+        self.trajectory_title = trajectory_title if trajectory_title is not None else ""
+        self.model_meta_data = (
+            model_meta_data if model_meta_data is not None else ModelMetaData()
+        )
```

Comment:
isn't this a longer way to set the defaults then doing it in the args list? Or am i missing something?

Reply:
yeah it's a bad idea to include mutable defaults in the args, so I've just done it in general. And actually I just remembered it's not needed for strings, but for ModelMetaData and any lists or dictionaries it's better to do it this way. Link from Na Hyung: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/. I'll update the strings tho

Reply:
Oh, I didn't realize a class would be considered mutable. Aren't you declaring a fresh instance of it by calling `ModelMetaData()` ? (this is just for my own understanding)

Reply:
yeah they're mutable https://stackoverflow.com/questions/12076445/are-user-defined-classes-mutable